staging/cert-manager-setup/ci/general-test-values.yaml

```diff
@@ -20,6 +20,15 @@ certificates:
     dnsNames:
       - example.com
       - www.example.com
+    ipAddresses:
+      - 127.0.0.1
+    keyAlgorithm: rsa
+    keyEncoding: pkcs8
+    keySize: 4096
+    organization:
```

Comment:
Lets name this as "organizations" so that its clear that multiple can be provided.

Reply:
I would like to stick to the CRDs key name 😬 
But we can add a comment ;)